cxx/pclean/schema_helper_test.cc

```diff
@@ -295,40 +290,35 @@ BOOST_AUTO_TEST_CASE(test_make_hirm_schema_only_final_emissions) {
   T_noisy_relation nr1 = std::get<T_noisy_relation>(tschema["Specialty"]);
   BOOST_TEST(nr1.is_observed);
   BOOST_TEST((nr1.emission_spec.emission == EmissionEnum::bigram_string));
-  // "School", "Physician"  moved to the front of the list.
-  expected_domains = {"School", "Physician", "City", "Practice", "Record"};
+  expected_domains = {"School", "Physician", "Record"};
   BOOST_TEST(nr1.domains == expected_domains, tt::per_element());
 
   BOOST_TEST(tschema.contains("School"));
   T_noisy_relation nr2 = std::get<T_noisy_relation>(tschema["School"]);
   BOOST_TEST(nr2.is_observed);
   BOOST_TEST((nr2.emission_spec.emission == EmissionEnum::bigram_string));
-  // "School" moved to the front of the list.
-  expected_domains = {"School", "City", "Practice", "Physician", "Record"};
+  expected_domains = {"School", "Record"};
```

Comment:
I think this should have "Physician" too, since each class along the path from the cleanly-observed class to Record (C*) should be an input to the noisy relation, according to my read of Model 5

Reply:
Can you file an issue for that?  I'd like to think about it some more and address it in a separate PR.  This PR has already expanded a bunch from its original intent.

Reply:
Sure, I can file a separate issue for Model 5.